### PR TITLE
Handle different sample rates

### PR DESCRIPTION
**Pull request by sornas**

### CODE REVIEW

File: src/asset/audio.rs [hunk 1]
```diff
@@ -1,69 +1,68 @@
...
-        let mut samples = self.samples.write().unwrap();
-        match data {
-            wav::BitDepth::ThirtyTwoFloat(data) => *samples = data,
-            _ => todo!("Only WAV containing floats are currently supported"),
-        }
+pub fn load_data(bytes: Vec<u8>) -> Samples {
+    let (header, data) = wav::read(&mut std::io::Cursor::new(bytes)).unwrap();
+    let data = match data {
+        wav::BitDepth::ThirtyTwoFloat(data) =>  data,
+        _ => todo!("Only WAV containing floats are currently supported"),
```
Comment (FredTheDino) [inline src/asset/audio.rs:62]: Aren't the 16 floats?

**Review by FredTheDino — COMMENTED** [review]
Haven't tried this either. But it looks good!

File: src/asset/audio.rs (hunk 1 above)
Comment (sornas) [reply src/asset/audio.rs:62]: https://docs.rs/wav/1.0.0/wav/bit_depth/enum.BitDepth.html

Comment (sornas) [reply src/asset/audio.rs:62]: These are the only variants we get. I guess they're cast to f32?

**Review by FredTheDino — APPROVED** [review]

**Pull request merged**